Cargo.toml

```diff
@@ -19,6 +19,8 @@ maintenance = { status = "passively-maintained" }
 
 [dependencies]
 griddle = { version = "0.5", optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
 
 [features]
 amortize = ["griddle"]
+with_serde = ["serde"]
```

Comment:
Why not just have the feature be called `serde`? I think that will just work without adding anything to `[features]`.

Reply:
It would probably be easy to have a feature called `serde` but when I tried `serde = ["serde"]`, cargo wasn't particularly pleased. However, I know that it (or something close enough) *can* be done. I just wasn't sure how.

Reply:
Just don't list it under `[features]` at all, and that should do the trick :) Optional dependencies implicitly create a feature by the same name.